package/kernel/linux/modules/ipmi.mk

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See https://www.gnu.org/licenses/gpl-2.0.txt for more information
+#
+
+IPMI_MENU:=IPMI system interface support
+
+define KernelPackage/ipmi-handler
+  SUBMENU:=$(IPMI_MENU)
+  TITLE:=IPMI top-level message handler
+  KCONFIG:=CONFIG_IPMI_HANDLER=m \
+           CONFIG_ACPI_IPMI=m \
+           CONFIG_IPMI_PANIC_EVENT=y \
+           CONFIG_IPMI_PANIC_STRING=y \
+           CONFIG_IPMI_DEVICE_INTERFACE=n \
+           CONFIG_IPMI_SI=n \
+           CONFIG_IPMI_SSIF=n \
+           CONFIG_IPMI_WATCHDOG=n \
+           CONFIG_IPMI_POWEROFF=n \
+           CONFIG_SENSORS_IBMAEM=n \
+           CONFIG_SENSORS_IBMPEX=n
+  FILES:=$(LINUX_DIR)/drivers/char/ipmi/ipmi_msghandler.ko
+  AUTOLOAD:=$(call AutoLoad,60,ipmi_msghandler)
+endef
+
+define KernelPackage/ipmi-handler/description
+  This enables the central IPMI message handler, required for IPMI to work.
+
+  IPMI is a standard for managing sensors in a system.
+  (e.g. temperature, voltage, power supply, etc.)
+endef
+
+$(eval $(call KernelPackage,ipmi-handler))
+
+
+define KernelPackage/ipmi-si
+  SUBMENU:=$(IPMI_MENU)
+  TITLE:=IPMI System Interface handler
+  DEPENDS:=+kmod-ipmi-handler
+  KCONFIG:=CONFIG_IPMI_SI=m
```

Comment:
This is packed as a kernel module. That "=m" for the kernel's .config is sort of implied.

Reply:
OK, will change that when pushing new version of PR.

Reply:
I fixed this in the latest version of PR